mix.exs

```diff
@@ -42,6 +46,9 @@ defmodule Cassette.Mixfile do
     ]
   end
 
+  @elixir_version Version.parse!(System.version())
```

Comment:
Duped line here! It's already at line 4!

Reply:
fixed :)